.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+            }
+            
+            return 'false';
+
+  reject-non-members:
+    needs: is-org-member
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && needs.is-org-member.outputs.member != 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reaction to rejected comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: 'confused'
+            })
+
+      - name: Comment PR (Rejected)
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `Sorry, only ${{ github.event.repository.owner.login }} members can run commands.`
+            })
+
+  acknowledge:
+    needs: is-org-member
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && needs.is-org-member.outputs.member == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reaction to triggered comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: 'eyes'
+            })
+
+  clean:
+    needs: is-org-member
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Clean previous comments
+        if: ${{ startsWith(github.event.comment.body, '/cmd') && contains(github.event.comment.body, '--clean') && needs.is-org-member.outputs.member == 'true' }}
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.listComments({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            }).then(comments => {
+              for (let comment of comments.data) {
+                console.log(comment)
+                if (
+                  ${{ github.event.comment.id }} !== comment.id &&
+                    (
+                      (
+                        (
+                          comment.body.startsWith('Command') || 
+                          comment.body.startsWith('<details><summary>Command') || 
+                          comment.body.startsWith('Sorry, only ')
+                        ) && comment.user.type === 'Bot'
```

Comment:
Why does it not delete all comments?

Reply:
Why should it delete user comments?

Reply:
Not user comments, but bot comments without one of these `startsWith`.

Reply:
Because perhaps there are other bots and automations that say something useful. It is aimed to clean only artefacts of itself